css/modules/styles.css

```diff
@@ -2156,24 +1989,29 @@ div.ui-memblist-usr .userVideos {
 
 #virtualclassCont.congrea #mainerrorContainer {
   position: absolute;
-  bottom: 50px;
-  left: 50%;
-  transform: translate(-50%);
+    left: 50%;
+    transform: translate(-50%, 0%);
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)